aerofiles/flarmcfg/writer.py

```diff
@@ -47,6 +47,16 @@ def format_longitude(self, value):
         return self.format_coordinate(
             value, default='00000000E', is_latitude=False)
 
+    def format_waypoints(self, SeeYou_task):
```

Comment:
I'd prefer it if the Flarm writer didn't know anything about SeeYou and how their tasks are structured.  This code should probably live in some sort of task converter module instead.